### DISCUSSION TITLE
how to config with https

### DISCUSSION THREAD

**Discussion by theone-daxia** [orchidsoftware/platform#2621] [Q&A] [answered]
how to config with https when login?

**Comment by tabuna** [comment, marked as answer]
Orchid does not regulate `http/https` in any way. This is not the responsibility of the package.

You can learn how Laravel regulates http/https through the `symfony/http-foundation` package. To determine whether a conversation is protected, it checks the value of `$_SERVER['HTTPS']`:


```bash
 `isSecure()` | `$_SERVER['HTTPS']`
```

This global variable should be set by your Apache/Nginx/Caddy/etc. web server.

If you don't have control over the web server, you can use a workaround by adding the following line to the `boot()` method in `AppServiceProvider`:

```php
\Illuminate\Support\Facades\URL::forceScheme('https');
```
